src/SDK/Metrics/Meter.php

```diff
@@ -133,25 +147,31 @@ public function createObservableGauge(string $name, ?string $unit = null, ?strin
         return new ObservableGauge($this->writer, $instrument, $referenceCounter, $destructors);
     }
 
-    public function createUpDownCounter(string $name, ?string $unit = null, ?string $description = null): UpDownCounterInterface
+    public function createUpDownCounter(string $name, ?string $unit = null, ?string $description = null, array $advisory = []): UpDownCounterInterface
     {
         [$instrument, $referenceCounter] = $this->createSynchronousWriter(
             InstrumentType::UP_DOWN_COUNTER,
             $name,
             $unit,
             $description,
+            $advisory,
         );
 
         return new UpDownCounter($this->writer, $instrument, $referenceCounter);
     }
 
-    public function createObservableUpDownCounter(string $name, ?string $unit = null, ?string $description = null, callable ...$callbacks): ObservableUpDownCounterInterface
+    public function createObservableUpDownCounter(string $name, ?string $unit = null, ?string $description = null, $advisory = [], callable ...$callbacks): ObservableUpDownCounterInterface
     {
+        if (is_callable($advisory)) {
```

Comment:
this is for BC, right? If so, is it worth documenting that it could be removed in a future breaking version?

Reply:
IMO we should wait with marking this for future removal until we actually plan to break BC / prepare for an API 2.0 release that changes the type to `array $advisory` (which I assume won't happen in the near future). Removal in the SDK can then happen in a minor version.